source/includes/aws/_objects.md

```diff
@@ -167,24 +230,24 @@ Download an object from a bucket in Amazon S3.
 
 > The above command returns the binary that represent the file downloaded.
 
-Required Query Parameters | &nbsp;
+Mandatory Query Parameters | &nbsp;
```

Comment:
<img width="300" alt="Screen Shot 2022-03-25 at 3 34 06 PM" src="https://user-images.githubusercontent.com/19736641/160189432-1a9c34b8-d0f6-41f4-8caf-9f42ef6c99b0.png">
confused why these are all mandatory when in the rest of the api docs we usually use required